src/main/java/org/opentripplanner/routing/api/request/request/filter/FilterPredicate.java

```diff
@@ -0,0 +1,18 @@
+package org.opentripplanner.routing.api.request.request.filter;
+
+import org.opentripplanner.transit.model.network.Route;
+import org.opentripplanner.transit.model.timetable.TripTimes;
+
+public interface FilterPredicate {
```

Comment:
This should be renamed to `TransitFilter`

Reply:
I renamed it